src/routes/simple/Simple.svelte

```diff
@@ -13,7 +13,7 @@
   <title>{data.title}</title>
 </svelte:head>
 
-<a href="/">&LeftArrow; Home</a>
+<a href={settings['$$internal'].serverPrefix}>&LeftArrow; Home</a>
```

Comment:
I know that we should not use `$$internal` here, but I don't know a better approach to do it